packages/robusta_events/lib/src/manager.dart

```diff
@@ -33,7 +33,7 @@ class DefaultEventManager implements EventManager {
   final Map<Type, EventStore<Event>> _stores = {};
 
   /// Get [EventStore] by [Event] type.
-  EventStore<E> _store<E extends Event>() {
+  EventStore<E> _getEventStore<E extends Event>() {
     final manager = _stores[E];
```

Comment:
Since the function is trying to retrieve Event from local stored map event, also the comments above the function definition stated getting EventStore by type Event. Therefore, I think renaming to _getEventStore is more accurate in this scenario. 

Please correct me if there is any misunderstanding.

Reply:
I agree with you, but I am not talk about the method name, I concerns the manager variable, in this context it should be named with store instead manager.

Reply:
Yup, that can be changed to store.